tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -33,3 +33,25 @@ def test__contains__(self):
         assert not isaspect(str)
         exc.match("<class 'str'> is not an "
                   'aspectclass or an instance of an aspectclass')
+
+    def test_get(self):
+        list_of_aspect = AspectList(
+            [Metadata.CommitMessage.Shortlog, Metadata.CommitMessage.Body])
+        assert list_of_aspect.get(Metadata.CommitMessage.Shortlog) is \
```

Comment:
These can be rewritten to not use pytest fixtures, but at this stage it isnt worth it.  So long as you are using normal unittest where possible, using pytest only when needing its advanced features is reasonable.